pkg/controller/perconaservermongodb/backup.go

```diff
@@ -186,3 +185,29 @@ func (r *ReconcilePerconaServerMongoDB) isBackupRunning(cr *api.PerconaServerMon
 
 	return false, nil
 }
+
+func (r *ReconcilePerconaServerMongoDB) updatePBMConfig(cr *api.PerconaServerMongoDB) error {
+	isRestoring, err := r.isRestoreRunning(cr)
+	if err != nil {
+		return fmt.Errorf("checking if restore running on pbm update: %w", err)
+	}
+
+	if isRestoring {
+		return nil
+	}
+
+	pbm, err := backup.NewPBM(r.client, cr)
+	if err != nil {
+		return fmt.Errorf("create pbm object: %w", err)
+	}
+
+	var stg api.BackupStorageSpec // getting the first and only storage from map
+	for _, stg = range cr.Spec.Backup.Storages {
+	}
+
+	if err = pbm.SetConfig(stg, cr.Spec.Backup.PITR); err != nil {
```

Comment:
can we please don't override the storage config here? just change the PITR enabled/disabled only if needed
storage should be configured by backup task

`github.com/percona/percona-backup-mongodb/pbm/config.go` has `confSetPITR("pitr.enabled", cr.Spec.Backup.PITR)` and `GetConfigVar("pitr.enabled")`

Reply:
If it's the first reconcile, we don't have any config to override. Can we set it here if it's not present?

And then update single config var (`pitr.enabled`)

Reply:
AFIK PITR cannot be enabled without a full backup.
If storage is empty - it means no full backup was made.
if no full backup made, it has no sense to try to enable PITR